### PR TITLE
IZ Parser

### PR DESCRIPTION
**Pull request by SuvorkinPavel**
Added parser for iz.ru/news

### CODE REVIEW

File: parsers/parser_iz.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
+import argparse
+import math
+import os
+import time
+
+import dateparser
+from selenium import webdriver
+from selenium.webdriver import ActionChains
```
Comment (AndRayt) [inline parsers/parser_iz.py]: Unused import. Need to remove.

File: parsers/parser_iz.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    @staticmethod
+    def __get_news_title(driver):
+        news_title = driver.find_element(By.CLASS_NAME, "node__cart__item__inside__info__title").text
+        return news_title
+
+    @staticmethod
+    def __get_news_text(driver):
+        try:
+            news_body = driver.find_element(By.CLASS_NAME, "text-article__inside").text
+        except:
```
Comment (AndRayt) [inline parsers/parser_iz.py]: PEP 8: E722 do not use bare 'except'
`except NoSuchElementException` ?

**Review by AndRayt — CHANGES_REQUESTED** [review]

File: parsers/parser_iz.py (hunk 1 above)
Comment (SuvorkinPavel) [reply parsers/parser_iz.py]: Removed in commit:
https://github.com/AndRayt/RussianNewsClassification/pull/4/commits/22534bc6cd424eaf7d0fdd67843f25a8117a430a

File: parsers/parser_iz.py (hunk 2 above)
Comment (SuvorkinPavel) [reply parsers/parser_iz.py]: Added in commit:
https://github.com/AndRayt/RussianNewsClassification/pull/4/commits/22534bc6cd424eaf7d0fdd67843f25a8117a430a

**Review by AndRayt — APPROVED** [review]
LGTM

**Pull request merged**